### PR TITLE
Pull Request Worker now collects pull_request_repo data

### PR DESCRIPTION
**Pull request by parthsharma2**
The Pull Request Worker now collects data for the pull_request_repo table

Signed-off-by: Parth Sharma <parth261297@gmail.com>

### CODE REVIEW

File: workers/pull_request_worker/pull_request_worker/worker.py [hunk 1]
```diff
@@ -790,6 +798,49 @@ def query_pr_comments(self, owner, repo, gh_pr_no, pr_id):
...
+            if new_pr_repo[0]['owner'] and 'login' in new_pr_repo[0]['owner']:
+                cntrb_id = self.find_id_from_login(new_pr_repo[0]['owner']['login'])
+            else:
+                cntrb_id = 1
+
+            pr_repo = {
+                'pr_repo_meta_id': pr_meta_id,
+                'pr_repo_head_or_base': pr_repo_type,
+                'pr_src_repo_id': new_pr_repo[0]['id'],
+                # 'pr_src_node_id': new_pr_repo[0]['node_id'],
```
Comment (sgoggins) [inline workers/pull_request_worker/pull_request_worker/worker.py]: @parthsharma2 : Are we going to add the src fork data in a different pull request?

File: workers/pull_request_worker/pull_request_worker/worker.py [hunk 2]
```diff
@@ -677,6 +677,10 @@ def query_pr_meta(self, head, base,  pr_id):
 
             self.pr_meta_id_inc = int(result.inserted_primary_key[0])
             self.results_counter += 1
+
+            if new_head[0]['repo']:
+                self.query_pr_repo(new_head[0]['repo'], 'head', self.pr_meta_id_inc)
```
Comment (sgoggins) [inline workers/pull_request_worker/pull_request_worker/worker.py]: @parthsharma2 : ... or, ignoring my other comment, is this where we are adding meta information for the source repo and the base repo? What table is the fork going in if so?

Comment (parthsharma2) [reply workers/pull_request_worker/pull_request_worker/worker.py]: The `query_pr_meta()` method adds the HEAD & BASE metadata to the `pull_request_meta` table. The repositories associated with HEAD & BASE are then added to the `pull_request_repo` table by the `query_pr_repo()` method.

File: workers/pull_request_worker/pull_request_worker/worker.py (hunk 1 above)
Comment (parthsharma2) [reply workers/pull_request_worker/pull_request_worker/worker.py]: If by src fork data you mean, the HEAD & BASE repos, then no. The `query_pr_repo()` method inserts the HEAD & BASE repos data into the `pull_request_repo` table.

Comment (sgoggins) [reply workers/pull_request_worker/pull_request_worker/worker.py]: @parthsharma2 : is that new? I don't see it populated (the head and base) in the current version.  Is this in the PR or is it supposed to already be there?

Comment (parthsharma2) [reply workers/pull_request_worker/pull_request_worker/worker.py]: The current version adds the head and base metadata to the `pull_request_meta` table. This PR adds the head and base repo data to the `pull_request_repo` table.

**Pull request merged**